functional_testing/zrobot_templates/framework/ovc_utils/templates/disk.yaml

```diff
@@ -1,4 +1,4 @@
-{% include 'node.yaml'%}
+{% include 'vdc.yaml'%}
```

Comment:
i think this should be changes .. as now we will have same vdc service written twice .. i guess